src/libraries/Common/tests/Tests/System/StringTests.cs

```diff
@@ -4038,6 +4081,32 @@ public static void Join_ObjectArray(string separator, object[] values, string ex
         {
             Assert.Equal(expected, string.Join(separator, values));
             Assert.Equal(expected, string.Join(separator, (IEnumerable<object>)values));
+            Assert.Equal(expected, string.Join(separator, (ReadOnlySpan<object>)values));
+        }
+
+        public static IEnumerable<object[]> Join_CharSeparator_ObjectArray_TestData()
+        {
+            yield return new object[] { '$', new object[] { }, "" };
+            yield return new object[] { '$', new object[] { new ObjectWithNullToString() }, "" };
+            yield return new object[] { '$', new object[] { "Foo" }, "Foo" };
+            yield return new object[] { '$', new object[] { "Foo", "Bar", "Baz" }, "Foo$Bar$Baz" };
+            //yield return new object[] { null, new object[] { "Foo", "Bar", "Baz" }, "FooBarBaz" };
```

Comment:
Why is this commented out?

Reply:
I forgot to delete it when I copied it from the `string separator` overload above. Thanks.